jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -338,7 +350,8 @@
  */
 ecma_number_t
 ecma_utf8_string_to_number (const lit_utf8_byte_t *str_p, /**< utf-8 string */
-                            lit_utf8_size_t str_size) /**< string size */
+                            lit_utf8_size_t str_size, /**< string size */
+                            bool is_infinity_check) /**< infinity check, default: false*/
```

Comment:
what is that "default" part? this is C, we have no default parameters. when should true be passed and when should false? there is a doc comment block before the function, please describe it there.